website/docs/r/app_configuration_key.html.markdown

```diff
@@ -106,7 +106,7 @@ resource "azurerm_app_configuration_key" "test" {
   key                    = "key1"
   type                   = "vault"
   label                  = "label1"
-  vault_key_reference    = azurerm_key_vault_secret.kvs.id
+  vault_key_reference    = azurerm_key_vault_secret.kvs.versionless_id
```

Comment:
IMO this can be left as is, the note you've added should be sufficient? I suppose it's a matter of preference whether you always want the latest version or not.

Reply:
Lots of people just copy and paste examples I for one did it and by accident discovered the pinned secret value, personally feel the example with `versionless_id` is the safer option of the two and will be less head-scratching for anybody in the future

Reply:
I guess it depends what is a better default, pinning to a key version, or just the key

Reply:
@katbyte happy to take guidance think I made my case for versionless_id as the default

Reply:
👍